src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -318,7 +295,7 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(fileUploadExerc
     }
 
     /**
-     * PUT /file-upload-exercises/{exerciseId}/re-evaluate : Re-evaluates and updates an existing fileUploadExercise.
+     * PUT file-upload-exercises/{exerciseId}/re-evaluate : Re-evaluates and updates an existing fileUploadExercise.
```

Comment:
```suggestion
     * PUT file-upload-exercises/{exerciseId}/re-evaluate : Re-evaluates and updates an existing FileUploadExercise.
```
There are many more locations with that (minor) problem.